ui/App/ProfileScreen/ProjectCardSquare.svelte

```diff
@@ -0,0 +1,81 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="typescript">
+  import type { Project } from "ui/src/project";
+
+  import { Avatar, Badge, CopyableIdentifier } from "ui/DesignSystem";
+  import ProjectAnchorPopover from "ui/App/OrgScreen/ProjectAnchorPopover.svelte";
+  import ProjectStats from "./ProjectStats.svelte";
+
+  export let project: Project;
+  export let isMaintainer: boolean | false;
```

Comment:
This is sufficient: 
```suggestion
  export let isMaintainer: boolean;
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2398/commits/68806eacd94a8c126e11add5bdb44b8d66e0bbe1